activities/Fototoon.activity/lib/tutorial.js

```diff
@@ -0,0 +1,121 @@
+define(["webL10n"], function (l10n) {
+//define([], function () {
+	var tutorial = {};
+
+	tutorial.start = function(language) {
+        
+        //l10n.language.code = "e";
+        //console.log("In tutorial lang: "+language);
+        //console.log("In tutorial lang: "+l10n.language.code);
+		var steps = [
+			{
```

Comment:
Try to be consistent about indentation in the file: should be TAB or white spaces but not both mixed

Reply:
Fixed indentation.